docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -1484,16 +1484,21 @@ EXECUTE y;                              -- uses MYSCHEMA;
 [[control_query_default_examples]]
 === Examples of CONTROL QUERY DEFAULT
 
-* Increase the cache refresh time for the histogram cache to two hours (7,200 minutes).
+* This example changes the maximum degree of parallelism to 2 for a query. The value must be less than the number of CPUs in the cluster.
 +
 ```
-CONTROL QUERY DEFAULT CACHE_HISTOGRAMS_REFRESH_INTERVAL '7200';
+SQL>CONTROL QUERY DEFAULT PARALLEL_NUM_ESPS '2';
+
+--- SQL operation complete.
 ```
 
-* Reset the CACHE_HISTOGRAMS_REFRESH_INTERVAL attribute to its initial value in the current process:
+* This example resets the `PARALLEL_NUM_ESPS` attribute to its system value in the current process, 
+in this case, the compiler calculates the number of ESPs to be used.

```

Comment:
Minor grammar nit: This is a run-on sentence. Suggestion: "This example resets ... in the current process. In this case, ..." (That is, break it up into two sentences.)